src/main/app/guards/onlyDefendantLinkedToClaimCanDoIt.ts

```diff
@@ -6,7 +6,7 @@ import { User } from 'idam/user'
 import { Claim } from 'claims/models/claim'
 import { FeatureToggles } from 'utils/featureToggles'
 
-export class IsDefendantInCaseGuard {
```

Comment:
Can we do the same to `IsClaimantInCaseGuard`?

Reply:
OK